test/unit/spec/RecoveryQuestion_spec.js

```diff
@@ -239,6 +242,30 @@ function (Okta, OktaAuth, Util, RecoveryQuestionForm, Beacon, Expect, Router,
           expect(test.form.errorMessage()).toBe('The recovery question answer did not match our records.');
         });
     });
+    itp('triggers an afterError event if there is an error submitting the answer', function () {
+      return setup()
+        .then(function (test) {
+          test.setNextResponse(resError);
+          test.form.setAnswer('4444');
+          test.form.submit();
+          return tick(test);
```

Comment:
here too... In general for the tests that we expect errors, let's use `Expect.waitForFormError`